include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -565,9 +544,9 @@ struct sycl::is_device_copyable<_ONEDPL_SPECIALIZE_FOR(oneapi::dpl::unseq_backen
 {
 };
 
-template <typename _BinaryOperator, typename _Size, typename _Range>
+template <typename _BinaryOperator, typename _Size>
 struct sycl::is_device_copyable<_ONEDPL_SPECIALIZE_FOR(oneapi::dpl::unseq_backend::__brick_reduce_idx, _BinaryOperator,
```

Comment:
I'm glad we are getting rid of the dependence of these structs on the `Ranges` and transferring that to the call operator instead.  It better avoids errors like this one where we were making device copyability depend on `_Range` which was wrong.